packages/types/src/Metadata.spec.js

```diff
@@ -1,19 +0,0 @@
-// Copyright 2017-2018 @polkadot/types authors & contributors
```

Comment:
Rather put this one back - it has benefit running as part of the test suite, I'd rather not lose it. (Even the logging is useful since it shows that it works and then the unique types as well. So just even visually it helps a _lot_ in debugging the Metadata)